metricflow-semantics/metricflow_semantics/specs/spec_classes.py

```diff
@@ -570,6 +570,7 @@ class MetricSpec(InstanceSpec):  # noqa: D101
     alias: Optional[str] = None
     offset_window: Optional[PydanticMetricTimeWindow] = None
     offset_to_grain: Optional[TimeGranularity] = None
+    default_granularity: Optional[TimeGranularity] = None
```

Comment:
The fields in the spec classes should be values that can't be inferred from the semantic manifest. From `get_default_granularity_for_metrics`, it seems like it is?

Reply:
Updated!